_pages/en/strategy-build-vs-buy.md

```diff
@@ -241,37 +331,37 @@ Whether the solution is written by a team within the Government of Canada, or by
 Proper security considerations must be taken prior to publishing code in the open, and teams are expected to work in tandem with, or acquire the necessary skills to, practice best practice while publishing code in the open.
 For further guidance on the use of Open Source, please reference to Annex 5, Use of Open Source.
 
-##### 5. Minimize the size, cost, and scale of work packages
+##### 5. Minimize the size, cost, and scale of work packages<!-- omit in toc -->
```

Comment:
remove from Appendix A (diagnostic). This appendix should only list problem statements (the diagnosis) that is leading us to make policy decisions (the strategy).